src/ORM/DataExtension.php

```diff
@@ -23,6 +24,7 @@ abstract class DataExtension extends Extension
      */
     public static function unload_extra_statics($class, $extension)
     {
+        Deprecation::notice('4.7.0', 'No longer used by internal code');
```

Comment:
Update the message to something standard (sounds like in this case "Will be removed without equivalent functionality")

Reply:
Updated